<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="DrawWidget" name="drawArea" native="true">
    <property name="geometry">
     <rect>
      <x>-170</x>
      <y>0</y>
      <width>1241</width>
      <height>611</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>84</width>
       <height>146</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnLimpar">
        <property name="text">
         <string>Limpar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCor">
        <property name="text">
         <string>Mudar Cor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBorracha">
        <property name="text">
         <string>Borracha</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboForma">
        <item>
         <property name="text">
          <string>Livre</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Linha</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Retângulo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Círculo</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTamanho">
        <property name="text">
         <string>Tamanho</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>728</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrawWidget</class>
   <extends>QWidget</extends>
   <header>drawwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
